.github/workflows/nodejs.yml

```diff
@@ -67,7 +67,17 @@ jobs:
         with:
           node-version: ${{ matrix.node-version }}
 
+      - name: Restore lerna cache
+        id: cache
+        uses: actions/cache@v2
+        with:
+          path: |
+            node_modules
+            */*/node_modules
+          key: ${{ runner.os }}-${{ hashFiles('**/yarn.lock') }}
+
       - name: Install dependencies
+        if: steps.cache.outputs.cache-hit != 'true'
```

Comment:
@evilebottnawi we're good now, we needed another step to skip install deps when cache is present